resources/public/api/conf/1.0/queryParameters.raml

```diff
@@ -10,6 +10,12 @@ traits:
         description: 'The tax year the data applies to, for example, 2018-19. The start year and end year must not span two tax years. No gaps are allowed, for example, 2017-19 is not valid. The minimum tax year is 2016-17.'
         example: "2017-18"
         required: false
+  taxYear201819:
```

Comment:
Not sure this is right? BF losses min is 2018-19 (agreed) but claims min is 2019-20.